src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/RuntimeEnvironmentTest.netstandard1.7.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Runtime;
+using System.Runtime.InteropServices; 
+using System.IO;
+using Xunit;
+
+namespace System.Runtime.InteropServices
+{
+    public static class RuntimeEnvironmentTest 
+    {
+        [Fact]
+        public static void RuntimeEnvironmentPosTest()
+        {
+            Assert.True(Directory.Exists(RuntimeEnvironment.GetRuntimeDirectory()));
+        }
+        
+        [Fact]
+        public static void RuntimeEnvironmentNegTest()
+        {
+             Type clsType   = typeof(RuntimeEnvironmentTest);
+             Assembly assem = clsType.Assembly;
+             Assert.True(!RuntimeEnvironment.FromGlobalAccessCache(assem));
+             
+             Assert.Throws<PlatformNotSupportedException>(() => RuntimeEnvironment.SystemConfigurationFile);
+             Assert.Throws<PlatformNotSupportedException>(() => RuntimeEnvironment.GetSystemVersion());
+             
+             Guid guid;
+             Assert.Throws<PlatformNotSupportedException>(() => RuntimeEnvironment.GetRuntimeInterfaceAsObject(guid,guid));
+             Assert.Throws<PlatformNotSupportedException>(() => RuntimeEnvironment.GetRuntimeInterfaceAsIntPtr(guid,guid));
+
+        }
```

Comment:
should i be adding [SkipOnTargetFramework(TargetFrameworkMonikers.Net46)] to the above fact ?